app/views/posts/index.html.erb

```diff
@@ -1 +1,41 @@
-<h1>Here's a list of Posts for a given User</h1>
+<%= stylesheet_link_tag 'users_index' %>
+<%= stylesheet_link_tag 'users_show' %>
+<%= stylesheet_link_tag 'posts_index' %>
+<section class='user-section'>
+    <div class='user-container'>
+        <%= image_tag(@user.photo) %>
+        <div class='user-info'>
+            <h2 class='username'><%= @user.name %></h2>
+            <p class='number-of-posts'>Number of posts: <%= @user.posts_counter %></p>
+        </div>
+    </div>
+        
+    <div class='user-bio'>
+        <h4>Bio</h4>
+        <p><%= @user.bio %></p>
+    </div>
+
+    <div class='user-posts'>
+        <h2>Posts</h2>
+        <% @posts.each do |post| %>
+            <div class='post-container'>
+                <h3><%= post.title %></h3>
+                <p><%= post.text %></p>
+                <div class='post-interactions'>
+                    <p>Comments: <%= post.comments_counter %></p>
+                    |
+                    <p>Likes: <%= post.likes_counter %></p>
+                </div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update your `app/views/posts/index.html.erb` page by ensuring that when clicking on each **Post** takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png)